### ISSUE TITLE
Mmol, R_gas, and Cp should not be independent

### ISSUE THREAD

**Issue by deitrr** [deitrr/THOR#5]
It appears, though, that Mmol is not used by the code at all. Maybe in Joao's private version it is. In any case, the two quantities are interdependent, so the user should only be able to set one or the other. Same goes for R_gas. Perhaps the code just needs to check that these quantities are all self-consistent.

**Comment by nabajour** [comment]
Agreed. All the options should be loaded from an external file and be checked for consistency before running the simulation.
